dataplane/linux/bpf_route_mgr.go

```diff
@@ -422,12 +423,8 @@ func (m *bpfRouteManager) applyRoutesDelta(oldRoutes map[routes.Key]routes.Value
 }
 
 func (m *bpfRouteManager) onRouteUpdate(update *proto.RouteUpdate) {
-	if update.Type != proto.RouteType_WORKLOADS_NODE {
-		log.WithField("type", update.Type).Debug("Route type we're not interested in, ignoring")
-		return
-	}
-
-	if update.Node == m.myNodename {
+	// FIXME combine local veth info with route info.
+	if update.DstNodeName == m.myNodename {
```

Comment:
Can I sensibly review the changes in this file yet?  I'm assuming it's OK to skip for now.

Reply:
Correct, skip for now